src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -148,6 +151,8 @@ function MoneyRequestView({
         (field: ViolationField, data?: OnyxTypes.TransactionViolation['data']): boolean => !!canUseViolations && getViolationsForField(field, data).length > 0,
         [canUseViolations, getViolationsForField],
     );
+    const noteTypeViolations = transactionViolations?.filter((violation) => violation.type === 'note').map((v) => ViolationsUtils.getViolationTranslation(v, translate));
```

Comment:
@Krishna2323 Is the violation type 'note' for such violations? What type do we use for violation messages? Can we reuse the existing method for violation messages? If you have any confusion, please do raise it on the issue.

Reply:
I would say `getViolationsForField('receipt')` will give us the violations we want. Those are the violations that we are going to show below ReceiptAudit component, right?
 ```
                {canUseViolations && <ViolationMessages violations={getViolationsForField('receipt')} />}
 ```
 
 @Krishna2323 Were you able to test those violations?

Reply:
@Krishna2323 The type for such violation is "notice"

```
[
    {
        "type": "notice",
        "name": "modifiedAmount",
        "data": null
    },
    {
        "type": "notice",
        "name": "modifiedDate",
        "data": null
    }
]
```